tests/test_cache_2p_sessions.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import numpy as np
+
+from viral.cache_2p_sessions import extract_frozen_wheel_chunks
+
+
+def test_extract_frozen_wheel_chunks_standard_case() -> None:
+    """Test with grab stopped automatically and everything in order"""
+    stack_lengths_tiff = np.array([27000, 10000, 27000])
+    sampling_rate = 10000
+    valid_frame_times = np.arange(0, 64000) * (sampling_rate / 30)
+    behaviour_times = np.arange(27100, 36900) * (sampling_rate / 30)
+
+    expected = (0, 26999), (36999, 63999)
+    result = extract_frozen_wheel_chunks(
+        stack_lengths_tiffs=stack_lengths_tiff,
+        valid_frame_times=valid_frame_times,
+        behaviour_times=behaviour_times,
+        sampling_rate=sampling_rate,
+    )
+    assert expected == tuple(tuple(int(x) for x in t) for t in result)
```

Comment:
```suggestion
    assert expected == result
```
seems to work for me